fares-data-build-tool/fdbt-aws
# feature: create MI athena queries

# Description

-   Adds cloudformation for athena named queries as well as a script for creating required 'views' within athena

# Testing instructions

-   Check the named queries work as expected when run (in the aws console, navigate to Athena -> Saved Queries and click on the title of a query to open it, then select 'Run query'

# Type of change


# Checklist:
